data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Recycle garbage"
+	name "Recycle garbage"
+	description "<origin> produces a lot of garbage that needs to be disposed of off-station. Transport <tons> of it to the industrial solid waste recycler on <destination>. Payment is <payment>.
+	job
+	repeat
+	cargo "Heavy Metals" 15 100
```

Comment:
Why is the garbage "heavy metals"? Incinerating any sort of heavy metals seems like a very bad idea...